app/src/main/java/com/example/stretchy/app/di/AppModule.kt

```diff
@@ -22,5 +23,5 @@ class AppModule {
     fun provideDataBase(application: Application): AppDatabase = Room.databaseBuilder(
         application.applicationContext,
         AppDatabase::class.java, AppDatabase.NAME
-    ).allowMainThreadQueries().build()
+    ).allowMainThreadQueries().addMigrations(MIGRATION_1_2).build()
 }
```

Comment:
I think we should remove allowMainThreadQueries. You've added coroutines so we dont need it.